tests/conftest.py

```diff
@@ -186,18 +186,8 @@ def application_executable(runtime: str) -> str:
 @fixture
 def gprofiler_exe(request: FixtureRequest, tmp_path: Path) -> Path:
     precompiled = request.config.getoption("--executable")
-    if precompiled is not None:
-        return Path(precompiled)
-
-    with chdir(PARENT):
```

Comment:
For the record, we are removing it here bcause this diverges from the pyi.Dockerfile. We follow the dockerfile build, not the logic here.